dev/release/utils-prepare.sh

```diff
@@ -155,4 +155,10 @@ update_versions() {
   find . -name "*.bak" -exec rm {} \;
   git add .
   popd
+
+  pushd "${ARROW_DIR}"
+  python "./ci/scripts/util_update_docs_versions.py" . "${version}" "${r_version}"
+  git add r/pkgdown/assets/versions.json
+  git add docs/source/_static/versions.json
+  popd
```

Comment:
I think that we should only update `versions.json`s only when `${type}` is `release`.